$NetBSD: patch-ac,v 1.1 2009/05/23 15:02:42 dholland Exp $

This patch fixes some stuff that breaks if you compile with assertions
and debugging on.

upstream: AFAIK upstream is currently comatose/dead

--- viewStats.c~	2000-01-16 13:47:02.000000000 -0500
+++ viewStats.c	2007-11-02 14:58:28.000000000 -0400
@@ -632,7 +632,7 @@
       break;
   
   /* There MUST be a slot */
-  D_Assert(iSlot >= MAX_PLAYERS, "Whoa, dude!  Bogus player somewhere?!");
+  D_Assert(iSlot < MAX_PLAYERS, "Whoa, dude!  Bogus player somewhere?!");
   
   /* Actually dump the data to the slot */
   STAT_RenderSlot(iSlot, iPlayer);
